docker/stellargraph-treon/Dockerfile

```diff
@@ -0,0 +1,40 @@
+# Image is needed anyway, so it's also used for downloading datasets
+FROM python:3.6-slim as base
+
+WORKDIR /data
+
+# Download datasets
+# hadolint ignore=DL3008
+RUN apt-get update \
+      && apt-get install --no-install-recommends -y curl unzip \
+      && curl -sLO "https://linqs-data.soe.ucsc.edu/public/lbc/cora.tgz" \
+      && tar --no-same-owner -xf cora.tgz \
+      && curl -sLO "https://linqs-data.soe.ucsc.edu/public/Pubmed-Diabetes.tgz" \
+      && tar --no-same-owner -xf Pubmed-Diabetes.tgz \
+      && curl -sLO "http://files.grouplens.org/datasets/movielens/ml-100k.zip" \
+      && unzip ml-100k.zip \
+      && curl -sLO "http://socialcomputing.asu.edu/uploads/1283153973/BlogCatalog-dataset.zip" \
+      && unzip BlogCatalog-dataset.zip
+
+# hadolint ignore=DL3006
+FROM stellargraph/stellargraph
+
+ENV JUPYTER_VERSION="1.0.0" \
+  PATH=${PATH}:/home/stellar/.local/bin
+
+# keep pip up to date for the end user
+# hadolint ignore=DL3013
+RUN pip install --upgrade pip --user \
+      && pip install --no-cache-dir jupyter=="${JUPYTER_VERSION}" treon seaborn --user  
```

Comment:
There are trailing spaces after `--user`

Reply:
Rule [DL3013](https://github.com/hadolint/hadolint/wiki/DL3013) ensure versions of pip packages are pinned. It might be worthwhile to pin `treon` and `seaborn` versions as `jupyter` is, and remove the ignore.

Reply:
Pinned Treon adn Seaborn versions.